www/src/pages/index.tsx

```diff
@@ -139,7 +139,6 @@ const HomeContent: React.FC = () => {
               })
             }
             frameBorder="0"
-            loading="lazy"
```

Comment:
We need this, makes a huge improvement in page load time and lighthouse score

Reply:
Ah i see

Reply:
Yeah, and sorry about keeping these PRs open haven't had a chance yet to dig back in.

Reply:
Thanks!